addons/binding/org.openhab.binding.smartmeter/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,43 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Smartmeter Binding
+Bundle-SymbolicName: org.openhab.binding.smartmeter;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .,
+ lib/jsml.jar,
+ lib/j62056.jar,
+ lib/rxjava-2.2.3.jar,
+ lib/reactive-streams-1.0.2.jar
+Import-Package: 
+ javax.measure,
+ javax.measure.format,
+ javax.measure.quantity,
+ javax.measure.spi,
+ org.apache.commons.codec,
```

Comment:
You seem to require apache commons codec only for the `Hex` class. Could you check whether ESH's [`HexUtils`](https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/util/HexUtils.java) can serve this purpose instead?

Reply:
Yes it serves the purpose. I'm changing it.